modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -84,6 +84,28 @@ $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceroutings.
 $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
 ----
 +
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspacetemplates.workspace.devfile.io
+----
++
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceoperatorconfigs.controller.devfile.io
+----
++
+[WARNING]
+====
+The DevWorkspace Operator ships custom resource definitions that make use of conversion webhooks. Failing to remove the CRDs above can cause issues on the cluster, as the steps below will remove the server that handles these conversion webhooks.
+====
++
+. Verify that all involved custom resource definitions are removed. The command below should return no results.
```

Comment:
**Suggestion**: "The command below... " --> "Note that on successful removal, the following command should not display any result."

Reply:
@sounix000 **Suggestion** that 'Note' as a word should be avoided. Can we retain 'Verify' at the start and the rest of the sentence should change as per your suggestion? In my opinion, "Note" should be used only when we're writing an actual "Note".

Reply:
Changed to 
> Verify that all involved custom resource definitions are removed. The following command should not display any result.

I think it's useful to mention the first part (custom resource definitions) to explain what is being checked.